TechAndCheck/zenodotus
# Remove Twitter to Hypatia

This offloads Twitter to Hypatia from here. We're doing this so that all scraping happens away from Zeno and we get a proper separation of concerns.

To test:
- `bundle install`
- `rails t`
- Launch and make sure Hypatia is running and try to archive a Twitter URL. This should kick off to Hypatia instead of running locally. Make sure you have the latest version of Hypatia
